live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/RequestFiltersIntegrationTest.java

```diff
@@ -0,0 +1,481 @@
+/*
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.hxi_connector.live_ingester.domain.usecase.e2e.repository;
+
+import static org.alfresco.hxi_connector.live_ingester.util.ContainerSupport.REQUEST_ID_PLACEHOLDER;
+
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import org.alfresco.hxi_connector.live_ingester.util.E2ETestBase;
+
+@SpringBootTest(properties = {"alfresco.filter.aspect.deny[0]=sc:secured",
+        "alfresco.filter.aspect.allow[0]=cm:versionable", "alfresco.filter.aspect.allow[1]=cm:author",
+        "alfresco.filter.aspect.allow[2]=cm:titled", "alfresco.filter.aspect.allow[3]=cm:preferences",
+        "alfresco.filter.aspect.allow[4]=cm:thumbnailModification", "alfresco.filter.aspect.allow[5]=cm:ownable",
+        "alfresco.filter.aspect.allow[6]=cm:auditable"})
```

Comment:
Why did you go for such a long list of allowed aspects? This is too many for me to remember while browsing the tests :yum:

Reply:
Sorry - some test setup tryouts leftover - making it smaller in the upcoming PR,